Rspec/specs/book_spec.rb

```diff
@@ -0,0 +1,48 @@
+require_relative '../../classes/book'
+require_relative '../../classes/item'
+
+describe Book do
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up: There is an error when I try to run the tests in this file. Kindly have a look and try to fix the error still this error is not fixed kindly fix this error :slightly_smiling_face: 

![Screenshot from 2023-08-11 00-21-31](https://github.com/Mahamadou-Nouridine/catalog-of-my-things/assets/45226848/dfadadf4-c7c2-4a4d-a16e-f810e669f6a8)